engine/swarm/secrets.md

```diff
@@ -45,18 +45,20 @@ encrypted. The entire Raft log is replicated across the other managers, ensuring
 the same high availability guarantees for secrets as for the rest of the swarm
 management data.
 
->**Warning**:
->Raft data is encrypted in Docker 1.13 and higher. If any of your
-Swarm managers run an earlier version, and one of those managers becomes the
-manager of the swarm, the secrets will be stored unencrypted in that node's Raft
-logs. Before adding any secrets, update all of your manager nodes to Docker 1.13
-to prevent secrets from being written to plain-text Raft logs.
+> **Warning**: Raft data is encrypted in Docker 1.13 and higher. If any of your
+> Swarm managers run an earlier version, and one of those managers becomes the
+> manager of the swarm, the secrets will be stored unencrypted in that node's
+> Raft logs. Before adding any secrets, update all of your manager nodes to
+> Docker 1.13 or higher to prevent secrets from being written to plain-text Raft
+> logs.
 {:.warning}
 
 When you grant a newly-created or running service access to a secret, the
-decrypted secret is mounted into the container in an in-memory filesystem at
-`/run/secrets/<secret_name>`. You can update a service to grant it access to
-additional secrets or revoke its access to a given secret at any time.
+decrypted secret is mounted into the container in an in-memory filesystem. The
+location of the mount point within the container defaults to
+`/run/secrets/<secret_name>`, but you can specify a custom location in Docker
+17.06 and higher. You can update a service to grant it access to additional
```

Comment:
Let's mention that giving an absolute target path is the way to specify a custom location; otherwise users may be confused about how to override the mount point.